src/services/case.ts

```diff
@@ -1,7 +1,7 @@
 import { UUID } from 'crypto';
 import { backendFetchJson } from './utils';
 
-const PREFIX_CASE_QUERIES = `${process.env.REACT_APP_API_GATEWAY}/case`;
+const PREFIX_CASE_QUERIES = import.meta.env.VITE_API_GATEWAY + '/case';
```

Comment:
why not keeping template form? not supported by vite/rollup?
```suggestion
const PREFIX_CASE_QUERIES = `${import.meta.env.VITE_API_GATEWAY}/case`;
```

Reply:
I prefer this form